geo/src/algorithm/euclidean_distance.rs

```diff
@@ -456,7 +456,8 @@ where
             }
             return mindist;
         }
-        if poly2.is_convex() || !self.is_convex() {
+        use super::is_convex::IsConvex;
+        if !poly2.exterior().is_convex() || !self.exterior().is_convex() {
```

Comment:
👍 I'll close #509

Reply:
Oh yeah, I figured it's a typo and fixed it as even the comments above the code mention that both should be convex.